Samagra-Development/workflow
# Dockerize workflow

Fixes #39 #27


It aims to dockerize all the services running in the workflow for easier deployments

- Form Manager
- Enketo
- React Wrapper

There are some code changes in the services so that they can be dockerized. I have also added pnpm-lock.yaml to all the services otherwise builds inside docker containers were failing. 

# How to test 
Open this PR on [Gitpod](https://gitpod.io/#https://github.com/Samagra-Development/workflow/pull/38) and check all flows e2e

# How to run locally
Relevant steps from gitpod can be run in sequence on a server
